crates/nix_health/src/check/home_manager.rs

```diff
@@ -0,0 +1,38 @@
+use nix_rs::info;
+use serde::{Deserialize, Serialize};
+use std::{env, path::PathBuf};
+
+use crate::traits::{Check, CheckResult, Checkable};
+
+#[derive(Debug, Default, Serialize, Deserialize, Clone)]
+#[serde(default)]
+pub struct HomeManager {}
+
+impl Checkable for HomeManager {
+    fn check(&self, _: &info::NixInfo, _: Option<&nix_rs::flake::url::FlakeUrl>) -> Vec<Check> {
+        let is_nix_store = get_shell_configuration_path(".zshrc").starts_with("/nix/store/")
```

Comment:
This method is hacky but can't figure out a better way to check if home manager is activated. 

<img width="1626" alt="Screenshot 2024-10-07 at 2 01 36 PM" src="https://github.com/user-attachments/assets/838b6c2e-4ac2-4a5a-a140-9aeec8e1b150">